### ISSUE TITLE
Only shows the download status of first download

### ISSUE THREAD

**Issue by haani-niyaz** [haani-niyaz/atlassian-admin-tools#28]
Does not show the results beyond the first download.

```
2017-05-08 10:27:04,332 - INFO - Downloading 1 of 2..
2017-05-08 10:27:04,333 - WARNING - sample.war already exists.
```

**Comment by haani-niyaz** [comment]
Fixed in #31

**Comment by haani-niyaz** [comment]
Fixed in #31.
